src/main/c/cext/ruby.c

```diff
@@ -1806,6 +1806,18 @@ int rb_proc_arity(VALUE self) {
 
 // Utilities
 
+int rb_tr_to_int_const(VALUE value) {
+  if(value == Qfalse) {
+    return Qfalse_int_const;
+  } else if(value == Qtrue) {
+    return Qtrue_int_const;
+  } else if(value == Qnil) {
```

Comment:
Needs a space between `if` and `(`, could you add a rule for it in `tool/lint.sh` ?